.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": [
+        "prettier",
+        "prettier/@typescript-eslint"
+    ],
+    "plugins": [
+        "@typescript-eslint",
+        "prettier"
+    ],
+    "rules": {
+        "prettier/prettier": [
+            "error",
+            {
+                "semi": false,
+                "trailingComma": "all",
+                "singleQuote": true
+            }
+        ],
+        "curly": [
+            "error",
+            "multi-line"
+        ],
+        "camelcase": 2
+    },
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "sourceType": "module"
+    }
+}
```

Comment:
This is my recommended eslint setup. Makes the code very nice and prevents annoying whitespace changes etc.